src/Components/BlogPost/BlogPostNavigation.js

```diff
@@ -37,7 +37,7 @@ const BlogPostNextLink = Scrivito.connect(({ currentBlogPost }) => {
   const [newerPost] = Scrivito.getClass("BlogPost")
```

Comment:
It now needs to be `const newerPost = Scrivito.getClass("BlogPost")`, since `first()` will already return only one item.